packs/spells/dragon-wings.json

```diff
@@ -29,7 +29,49 @@
             "value": ""
         },
         "requirements": "",
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "Spend 2 Focus Points on Dragon Wings",
+                "option": "dragon-wings:2-focus-points",
+                "placement": "spellcasting",
+                "toggleable": true
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:slug:dragon-wings",
+                    "dragon-wings:2-focus-points"
+                ],
+                "property": "focus-point-cost",
+                "value": 1
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "label":"Heightened",
+                "predicate": [
+                    "item:slug:dragon-wings",
+                    {
+                        "gte":
+                        [
+                            "{item|level}",
+                            8
+                        ]
+                    }
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Dragon Wings (10 minutes)]"
+                    }
+                ]
+            }
```

Comment:
On one hand this is a neat way to approach this, on the other it will make migrating away this effect when we get duration alteration a huge pain.